macaroons/constraints_test.go

```diff
@@ -67,8 +68,8 @@ func TestTimeoutConstraint(t *testing.T) {
 	}
 
 	// Finally, check that the created caveat has an
-	// acceptable value
-	if strings.HasPrefix(string(testMacaroon.Caveats()[0].Id),
+	// acceptable value.
+	if !strings.HasPrefix(string(testMacaroon.Caveats()[0].Id),
```

Comment:
@guggero Does this change make sense?

Reply:
Yeah, this was indeed not correct. One of my very first PRs and it was quite wrong :see_no_evil: 
Turns out `string(intvalue)` creates a rune and not the string representation.

Reply:
While we're at it, can we also fix the formatting in this file? Looks like some function calls aren't in our preferred style and some lines are too long.